### DISCUSSION TITLE
How to mass convert Gliffy to draw.io?

### DISCUSSION THREAD

**Discussion by Mannshoch** [jgraph/drawio#4207] [Q&A]
I there a way to mass convert Gliffy to Drawio?

**Comment by davidjgraph** [comment]
Where are the gliffy files stored?

**Reply by Mannshoch** [reply]
As attachment per confluence page.

**Reply by davidjgraph** [reply]
https://www.drawio.com/doc/faq/mass-import-gliffy-confluence-cloud

**Reply by Mannshoch** [reply]
What if you have an on-premise confluence? As far I know that only work with a cloud.

**Reply by davidjgraph** [reply]
https://www.drawio.com/doc/faq/mass-import-gliffy-confluence-server

**Reply by Mannshoch** [reply]
OK, I checked the case again. The Problem is we do not have Draw.io installed on Confluence and also could not install it because the appstore do not allows it, our Confluence is two years old because we didn't migrated to the Confluence Datacenter edition.

**Reply by davidjgraph** [reply]
Then it's not possible to mass convert if you cannot install the software that performs the conversion. You could manual convert gliffy diagrams, one by one, but not on that server.

**Reply by Mannshoch** [reply]
Do you have apackage of the addin for server?
https://marketplace.atlassian.com/apps/1210933/draw-io-diagrams-whiteboards?tab=versions&hosting=server
If I could download it I may able to install it amanually.

**Reply by davidjgraph** [reply]
We're not the company that runs the atlassian integrations, I would suggest to try atlassian community.
